manifests/components/images.json

```diff
@@ -9,5 +9,6 @@
     "kibana": "bitnami/kibana:6.7.2-r1",
     "nginx-ingress-controller": "bitnami/nginx-ingress-controller:0.24.1-r4",
     "oauth2_proxy": "bitnami/oauth2-proxy:3.1.0-r1",
+    "curator": "bobrik/curator:5.7.6",
```

Comment:
Would be great if you sort this entry in the list, it makes it easy to parse the list.

Reply:
Addressed